README.md

```diff
@@ -21,13 +21,13 @@ simulators, and hardware for iOS and Android.
 
 ## Quick-start
 
-A brief summary of the [getting started guide](https://flutter.io/docs/development/tools/ide):
+A brief summary of the [getting started guide](https://flutter.dev/docs/development/tools/ide):
 
-- install the [Flutter SDK](https://flutter.io/docs/get-started/install)
+- install the [Flutter SDK](https://flutter.dev/docs/get-started/install)
 - run `flutter doctor` from the command line to verify your installation
 - ensure you have a supported [IntelliJ development environment](https://www.jetbrains.com/idea/download), either:
   - IntelliJ 2018.1 or more recent, Community or Ultimate Edition, or
-  - Android Studio 3.2 (note: Android Studio Canary versions are generally _not_ supported)
+  - Android Studio 3.2 or later (note: Android Studio Canary versions are generally _not_ supported)
```

Comment:
Not for this PR, but we should likely remove explicit version references, and just say something like the latest stable versions.

Reply:
Good idea. Is there a different criteria for development of this feature versus using it?

Reply:
Both flows just need a reasonably recent version of the IDEs (we have more detailed setup for contributors here: https://github.com/flutter/flutter-intellij/blob/master/CONTRIBUTING.md).